docs/docs/architecture/SR-logo.svg

```diff
@@ -1 +1,44 @@
.67A17,17,0,0,1,260.11,107.29Z"/><path class="cls-1" d="M289.34,102V88h-3.52V82.39h3.52V72.9l6.56-.7V82.39h8.37V88H295.9v13.36c0,2.92.88,4,2.75,4a9.63,9.63,0,0,0,4.22-1.23l1.7,5.68a13.29,13.29,0,0,1-7.27,1.7C292.38,111.45,289.34,108.34,289.34,102Z"/><path class="cls-1" d="M326.53,82v7.33c-.76-.06-1.52-.12-2-.12-5,0-7.79,3.57-7.79,8.26V111h-6.56V82.39h6.56v4.34h.06a11,11,0,0,1,8.9-4.75Z"/><path class="cls-1" d="M357.87,82.39l-15.7,40.19h-6.91l5.15-13.24-11.6-26.95h6.91l7.91,19.34L351,82.39Z"/><path class="cls-1" d="M42.31,47.78c-6.26,0-13.76-2.73-21.62-11.58-5.92-6.66-12.07-10-18.28-10H2.28a2.28,2.28,0,0,1,0-4.55h.15c7.57,0,14.86,3.89,21.68,11.56,10.14,11.42,19.22,11,25.05,8.69a31.57,31.57,0,0,0,11.51-8.69c6.82-7.67,14.12-11.56,21.69-11.56S97.2,25.5,104,33.17c5.92,6.66,12.07,10,18.28,10a2.28,2.28,0,0,1,0,4.56c-7.57,0-14.87-3.9-21.68-11.57-5.92-6.66-12.07-10-18.28-10S70,29.54,64.06,36.2a36,36,0,0,1-13.22,9.88A22.84,22.84,0,0,1,42.31,47.78Z"/><path class="cls-1" d="M41.54,134.19c-10.65,0-17.76-7.47-22.49-12.43-1.14-1.19-2.12-2.22-3-3a2.28,2.28,0,0,1,3-3.43c1,.87,2,2,3.25,3.23,4.49,4.71,10.64,11.16,19.54,11h.17c9,.14,15.08-6.3,19.57-11,1.21-1.27,2.26-2.37,3.26-3.24a2.28,2.28,0,1,1,3,3.43c-.85.73-1.83,1.76-3,3-4.79,5-12,12.6-22.91,12.42h-.47Z"/><path class="cls-1" d="M66.38,19.37a2.26,2.26,0,0,1-1.5-.56c-1-.87-2-2-3.25-3.24C57.14,10.86,51,4.41,42.09,4.56h-.17c-9-.15-15.08,6.3-19.57,11-1.21,1.27-2.26,2.37-3.26,3.24a2.28,2.28,0,0,1-3-3.43c.85-.74,1.83-1.76,3-3C23.84,7.41,31.05-.18,42,0h.09C52.92-.19,60.14,7.41,64.92,12.43c1.14,1.19,2.12,2.22,3,3a2.28,2.28,0,0,1-1.5,4Z"/><path class="cls-1" d="M82.48,69.43C71.84,69.43,64.72,62,60,57c-1.13-1.19-2.11-2.22-3-3a2.28,2.28,0,0,1,3-3.43c1,.87,2,2,3.26,3.24,4.49,4.71,10.64,11.15,19.54,11H83c8.94.16,15.08-6.3,19.57-11,1.21-1.27,2.26-2.37,3.26-3.24a2.28,2.28,0,0,1,3,3.43c-.84.74-1.82,1.76-3,3-4.78,5-12,12.61-22.9,12.43h-.48Z"/><path class="cls-1" d="M2.41,69.37H2.26a2.28,2.28,0,0,1,0-4.55h.13c6.21,0,12.36-3.38,18.28-10C27.39,47.23,34.57,43.34,42,43.21a2.28,2.28,0,0,1,.08,4.56c-6.12.1-12.17,3.48-18,10C17.27,65.48,10,69.37,2.41,69.37Z"/><path class="cls-1" d="M42.31,91c-6.26,0-13.76-2.73-21.62-11.58a43,43,0,0,0-4.21-4.14,2.28,2.28,0,0,1,2.95-3.48,47.77,47.77,0,0,1,4.66,4.59c10.14,11.42,19.22,11,25.05,8.68a31.42,31.42,0,0,0,11.51-8.68c6.82-7.67,14.12-11.56,21.69-11.56S97.2,68.71,104,76.38c5.92,6.66,12.07,10,18.28,10a2.28,2.28,0,0,1,0,4.56c-7.57,0-14.87-3.9-21.68-11.57-5.92-6.66-12.07-10-18.28-10S70,72.75,64.06,79.41a36,36,0,0,1-13.22,9.88A22.84,22.84,0,0,1,42.31,91Z"/><path class="cls-1" d="M82.34,112.58c-7.57,0-14.87-3.89-21.69-11.57a31.64,31.64,0,0,0-11.51-8.68C43.31,90,34.23,89.6,24.09,101c-6.86,7.73-14.22,11.62-21.83,11.57a2.28,2.28,0,0,1,0-4.55h.13c6.21,0,12.36-3.38,18.28-10,11.89-13.39,23-12.77,30.15-9.89A36.16,36.16,0,0,1,64.06,98c5.91,6.66,12.06,10,18.28,10s12.36-3.38,18.28-10a46.64,46.64,0,0,1,4.83-4.74,2.28,2.28,0,0,1,2.93,3.5A41.34,41.34,0,0,0,104,101C97.2,108.69,89.91,112.58,82.34,112.58Z"/></g></g></svg>
```

Comment:
We need to have docs from the get go... do not delete docs.. as that is primarily the design of stream-registry moving forward. `<BLOCK MERGE/>`